README.md

```diff
@@ -56,6 +56,10 @@ Sample output:
 
 The Web application and API Documentation can be accessed through te given URLs.
 
+If you need the Web application to be accessible from remote machines, set the host and port fields in the 
```

Comment:
This is very useful, because without this, users would get irritated (not knowing where's the problem)

### Suggestion
Beautify with Github markdown styling: \`foobar\`
 `host` `port` `[web_app.server]`
`~/.config/TensorHive/main_config.ini`

In the long run, we should also build a whole section dedicated for such descriptions (aka tips&tricks/common issues) under **Configuration** section or just put every one of them directly into `.ini` file.

Reply:
beautified in following commit